CHANGELOG.md

```diff
@@ -4,6 +4,75 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 0.5.0 - 2018-12-03
+
+### Added
+
+- We now push the ingress custom resource defintion and initial
+  configuration, allowing the ingress operator to configure itself
+  without referencing the deprecated `cluster-config-v1` resource.
+
+### Changed
+
+- Pull secret documentation now points to
+  [try.openshift.com](https://try.openshift.com) for pull-secret
+  acquisition, instead of pointing at `account.coreos.com`.  Users
+  will need to update their pull secrets.
+- If the automatic bootstrap teardown (which landed in 0.4.0) times
+  out waiting for the `bootstrap-complete` event, the installer exits
+  with a non-zero exit code.  We had ignored watcher timeouts in 0.4.0
+  due to concerns about watcher robustness, but the current watcher
+  code has been reliable in our continuous integration testing.
+- The hard-coded `quay.io/coreos/bootkube` dependency has been
+  replaced by the new [cluster-bootstrap][] image, which is referenced
+  from the release image.
+- The etcd service now uses [selectors][kube-selector] to determine
+  the pods it exposes, and the explict etcd endpoints object is gone
+  (replaced by the one Kubernetes maintains based on the selector).
+- On AWS, both masters and worker have moved from t2.medium nodes
+  m4.large nodes (more on AWS instance types
+  [here][aws-instance-types] to address CPU and memory constraints.
+- On AWS, master volume size has been bumped from 30 GiB to 120 GiB to
+  increase our baseline performance from on [gp2's sliding IOPS
+  scale][aws-ebs-gp2-iops] from the 100 IOPS floor up to 360 IOPS.
+  Volume information is not currently supported by [the cluster-API
+  AWS provider's
+  `AWSMachineProviderConfig`][cluster-api-provider-aws-012575c1-AWSMachineProviderConfig],
+  so this change is currently limited to masters created by the
+  installer.
+- On Openstack, we now validate cloud, region, and image-name user
+  input instead of blindly accepting entries.
+- On libvirt, we pass ignition information for masters and workers via
```

Comment:
s/ignition/Ignition/

Reply:
> s/ignition/Ignition/

Oops.  Fixed with dcb3909 -> c2263f6, which also rebases onto master to sit on top of #775 (although I've still left that fix undocumented, since it doesn't seem like a big deal).